app/controllers/subscriber_lists_controller.rb

```diff
@@ -35,6 +35,7 @@ def subscriber_list_params
       slug: slugify(title),
       url: params[:url],
       description: (params[:description] || ""),
+      list_group_id: params[:list_group_id],
```

Comment:
Do we need to prefix this with `list_` given that it's on the subscriber list model? It feels like `group_id` would be enough.

Reply:
I've made this change now.